server/src/test/java/io/spine/examples/shareaware/server/investment/SharesPurchaseTest.java

```diff
@@ -289,10 +273,10 @@ void cancelMoneyReservation() {
         @Test
         @DisplayName("which emits the `SharesPurchasedFailed` event after error in the Shares Market")
         void sharesPurchaseFailed() {
-            Wallet wallet = setUpReplenishedWallet(context());
-            PurchaseShares command = purchaseSharesFor(wallet.getId()
-                                                             .getOwner());
-            SharesPurchaseFailed expected = sharesPurchaseFailedAsResultOf(command);
+            var wallet = setUpReplenishedWallet(context());
+            var command = purchaseSharesFor(wallet.getId()
```

Comment:
Please search for this
```java
wallet.getId()
      .getOwner()
``` 
repeated for many times, and please do something with it.